scripts/create-movies/create-movies.js

```diff
@@ -72,56 +138,412 @@ dotenv.config();
+          // no need to update the keyword, because in Movies.csv, there is no handicap or kwType listed
+          return;
+        }
+
+        // create a new keyword
+        return await axios.post(
+          apiUri,
+          {
+            query: `
+              mutation CreateKeyword($data: KeywordCreateInput!) {
+                createKeyword(data: $data) {
+                      id
+                      name
+                  }
+              }
+            `,
+            variables: {
+              data: {
+                name: keyword,
+              },
+            },
+          },
+          {
+            headers: {
+              Cookie: `keystonejs-session=${sessionToken}`,
+            },
+          }
+        );
+      });
+
+      createReadStream("./scripts/create-movies/Movies.csv")
+        .pipe(csv())
+        .on("data", async (movie) => {
+          // get an array of all the keywords for a particular movie
+          const currentMovieKeywords = movie["Keywords"]
+            .split(",")
+            .map((keyword) => keyword.trim());
+
+          for (const keyword of currentMovieKeywords) {
+            keywordPromises.push(limitedHandleKeywords(keyword));
+          }
+        })
+        .on("end", async () => {
+          try {
+            console.log("Parsing keywords from Movies.csv. Please wait...");
+            await Promise.all(keywordPromises);
+            console.log(
+              `Successfully parsed ${keywordPromises.length} keywords!`
+            );
+            resolve();
+          } catch (error) {
+            reject(error);
+          }
+        })
+        .on("error", (error) => reject(error));
+    });
+  };
+
+  /* ----------------------------- handle movies ----------------------------- */
+  const seedMoviesFromMoviesCSV = () => {
+    return new Promise((resolve, reject) => {
+      const moviePromises = [];
+      const limitedHandleMovie = throat(10, async (movie) => {
+        const checkedMovie = await movieExists(
+          movie["Title"],
+          movie["Release Year"],
+          sessionToken
+        );
+        if (checkedMovie) {
+          const currentMovieKeywords = movie["Keywords"]
+            .split(",")
+            .map((keyword) => {
+              return { name: keyword.trim() };
+            });
+          // update the movie
+          return await axios
+            .post(
+              apiUri,
+              {
+                query: `
+            mutation UpdateMovie($where: MovieWhereUniqueInput!, $data: MovieUpdateInput!) {
+              updateMovie(where: $where, data: $data) {
+                    id
+                    title
+                }
+            }
+            `,
+                variables: {
+                  where: { id: checkedMovie.id },
+                  data: {
+                    description: movie["Description"],
+                    handicap: parseInt(movie["Handicap"]),
```

Comment:
We can remove `handicap` as we're going to update that to a virtual field, seen [here](https://github.com/jahorwitz/slopopedia-api/pull/18).

Reply:
Ok! I will remove the handicap then. Thank you!

Reply:
@tomershvueli I just wanted to mention that with this handicap line removed, movies still have a handicap, but it's just set to a default of 0. So, now all movies in the database will have a 0 handicap. But, I made the change.

Reply:
This is fine. The other PR I linked removes that field completely.